CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -135,7 +149,7 @@ def format_dynamodb_item(self):
         for key in self.ITEM_TYPES:
             val = getattr(self, key)
             if val is not None:
-                fridge_item[key] = {self.ITEM_TYPES[key]: val}
+                fridge_item[key] = {self.ITEM_TYPES[key]: str(val)}
```

Comment:
nit: is the type casting to `str` required?

Reply:
Yes, AWS takes in numbers are string. You get an error otherwise